applications/StructuralMechanicsApplication/custom_processes/shell_to_solid_shell_process.cpp

```diff
@@ -180,8 +181,10 @@ void ShellToSolidShellProcess<TNumNodes>::Execute()
     // Reorder again all the IDs
     ReorderAllIds();
 
-//     // We initialize the new elements
-//     InitializeElements();
+    // We initialize the new elements
+    if (mThisParameters["initialize_elements"].GetBool()) {
```

Comment:
the behaviour must be the same for all the elements...

and the Initialize MUST be called

Reply:
The thing is that this is called just after reading the model part the initialization will be called later, and the initialization should be called just once

Reply:
I understand, however the user should not configure this. It should be done correctly for everyone...

Reply:
As i recall the problem is that the ConstitutiveLaws need to initialized to be able to call the "Check" function

Reply:
When @jginternational finishes with the release we will discuss how to implement this in the interface. Even if the prismatic solid shell directly (without extruding the shell) the process to compute the neighbours should be called before the Initialize (there is a check and the element crashes without neighbours). All this will be put together, and the user will not have to worry about this